frontend/src/app/types/user.ts

```diff
@@ -0,0 +1,5 @@
+export class User {
+  id: number;
+  email: string;
+  nickname: string;
```

Comment:
https://github.com/swsnu/swpp18-team5/wiki/Model-Design#user

It should be `username`, not `nickname`.